diff --git a/lib/module/Header/HeaderBackground.js b/lib/module/Header/HeaderBackground.js
index 8d5aa8689df4720f5f03189f21d5d4fe2e241a24..892f5558abfc7834fc2f53f8b4f3282554dc12a8 100644
--- a/lib/module/Header/HeaderBackground.js
+++ b/lib/module/Header/HeaderBackground.js
@@ -38,6 +38,10 @@ const styles = StyleSheet.create({
           height: StyleSheet.hairlineWidth
         }
       },
+      web: {
+        borderBottomWidth: StyleSheet.hairlineWidth,
+        boxShadow: '0px 1px 3px rgba(0, 0, 0, 0.06)'
+      },
       default: {
         borderBottomWidth: StyleSheet.hairlineWidth
       }
